RELEASE_NOTES.rst

```diff
@@ -1,3 +1,121 @@
+coala 0.10 - PolarBear
+=======================
+
+::
+
+
+                       `++-    -o+`                   
+                -oo:  :yhho    ohhy:  :oo.            
+                :hhhoohhh+      +hhhoohhy:            
+            ``.--shhhhhy:        /yhhhhho--..`        
+           +hhhhhhhhhhh+          ohhhhhhhhhhh/       
+           `/+/////+hhhh/        +hhhy+///////`       
+                    -yhhhs     hshhhy-                
+       .os/           hhhhy-  -yhhhh           +ss.   
+       .yhho           shhho``ohhhs          `ohhy`   
+        -yhhs`          +hhhsshhh+          `shhy-    
+      .::shhhs++/+yhy////shhhhhhs////yhy++++yhhho::-  
+     /hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh/ 
+      -::shhhs++++yhy////shhhhhhs          +shhhs::.  
+        -yhhs`          +hhhsshhh+          `shhy-    
+       .yhho`          shhho``ohhhs           ohhy.   
+       .ss+           hhhhy   -yhhhh           /so.   
+                    -yhhhs      shhhy-                
+           `///////+yhhh+        /hhhh+/////+/`       
+           /hhhhhhhhhhho          +hhhhhhhhhhh+       
+            `..--ohhhhhy/        :yhhhhhs--.``        
+                :hhhoohhh+      +hhhoohhh:            
+                -oo:  :yhho    ohhy:  :oo-            
+                       `+o-    -++`
+
+For this release we had 168 commits contributed by 66 unique contributors
+over 2.5 months
+
+The name of this release is PolarBear to reflect changes we have made to our
+release process. We have implemented a release freeze before all releases to
+to serve as a testing period for all staged changes. This should also help
+us keep releases on schedule.
+
+This release was an enhancement release which brings the usual slew of
+enhancements to documentation and api. We have started preparing the codebase
```

Comment:
use "improvements", otherwise this sentence is really double-doubly ;)

"This **release** was an **enhancement release** which brings the usual slew of **enhancements**"